lib/tokenize.cpp

```diff
@@ -1145,6 +1151,7 @@ void Tokenizer::simplifyTypedef()
                 typedefInfo.lineNumber = typedefToken->linenr();
                 typedefInfo.column = typedefToken->column();
                 typedefInfo.used = t.second.isUsed();
+                typedefInfo.isFunctionPointer = isFunctionPointer(typedefToken->next(), t.second.endToken());
```

Comment:
I would prefer `Token::Match(t.second.nameToken(), "%name% ) (")` . The `TypedefSimplifier::nameToken()` must be added then also.